R/templates.R

```diff
@@ -2,14 +2,16 @@ find_template <- function(type,
                           name,
                           ext = ".html",
                           templates_dir = NULL,
-                          bs_version = 3) {
+                          bs_version = 3,
```

Comment:
I wonder if it's now the time to bite the bullet and pass `pkg` to this function?

Reply:
I agree, I'll add that change